### PR TITLE
fix: Update configs payment variable for Stripe webhook endpoint

### PR DESCRIPTION
**Pull request by julianajlk**
[REV-3238](https://2u-internal.atlassian.net/browse/REV-3238).

Another attempt to get the config variables right in Stage/Prod for the Stripe webhook endpoint. 

🕵️ @pshiu pointed out that Ansible appears to change the names of the environment variables under the hood ([GitHub link](https://github.com/openedx/ecommerce/blob/a3af1cb7ff3a7aef3afbac6ed9bf617fe0d7d4ac/ecommerce/extensions/payment/processors/__init__.py#L75-L86)), and that sometimes we use `ECOMMERCE_PAYMENT_PROCESSOR_CONFIG` (like in [base.py](https://github.com/openedx/ecommerce/blob/a3af1cb7ff3a7aef3afbac6ed9bf617fe0d7d4ac/ecommerce/settings/base.py#L777) where I originally got that from) and other times we use `PAYMENT_PROCESSOR_CONFIG` (in [_oscar.py](https://github.com/openedx/ecommerce/blob/a3af1cb7ff3a7aef3afbac6ed9bf617fe0d7d4ac/ecommerce/settings/_oscar.py#L144)). 

**This PR:**
- Updates getting the variables from `ECOMMERCE_PAYMENT_PROCESSOR_CONFIG` to `PAYMENT_PROCESSOR_CONFIG`
- Renaming `endpoint_secret` to more specific `webhook_endpoint_secret`. This is also being updated in `edx-internal` https://github.com/edx/edx-internal/pull/7763
- Adds `webhook_endpoint_secret` to some files that were missing

### CODE REVIEW

File: ecommerce/settings/test.py [hunk 1]
```diff
@@ -97,6 +97,7 @@
             'proxy': None,
             'publishable_key': 'fake-publishable-key',
             'secret_key': 'fake-secret-key',
+            'webhook_endpoint_secret': 'fake-secret-key',
```
Comment (pshiu) [inline ecommerce/settings/test.py]: Nit:

```suggestion
            'webhook_endpoint_secret': 'fake-webhook-endpoint-secret',
```

File: ecommerce/settings/test.py [hunk 2]
```diff
@@ -136,6 +137,7 @@
...
```
Comment (pshiu) [inline ecommerce/settings/test.py]: Nit:

```suggestion
            'webhook_endpoint_secret': 'fake-webhook-endpoint-secret',
```

**Review by pshiu — APPROVED** [review]
Good stuff!

File: ecommerce/settings/test.py (hunk 1 above)
Comment (julianajlk) [reply ecommerce/settings/test.py]: Good one, thanks Phil, corrected

**Pull request merged**